components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -4254,7 +4254,7 @@ document.observe("xwiki:dom:loaded", function() {
 
   widgets.FuzzyDatePickerDropdown = Class.create({
     initialize : function(options) {
-      this.span     = new Element('span');
+      this.span     = new Element('span', {"tabIndex":"-1"});
```

Comment:
I remember testing this in FF on Linux without any <span> elements, with just a <select> - it still takes two tabs iff a value has just been selected (one tab closes drop-down, another navigates to the next box); without "tabindex" it still took two tabs even from a closed drop-down, but only one tab with "tabindex" fix... Maybe a more complicated JavaScript solution is necessary to manually handle tab key presses and manually navigate to the next box?